### PR TITLE
CRM-19853 replace interval controls with numeric

### PR DESCRIPTION
**Pull request by konadave**
Replace the interval select controls with options numbered 1-30 with numeric controls.

---

 * [CRM-19853: Change interval select controls in Scheduled Reminders with number controls](https://issues.civicrm.org/jira/browse/CRM-19853)

### CODE REVIEW

File: CRM/Admin/Form/ScheduleReminders.php [hunk 1]
```diff
@@ -161,10 +161,10 @@ public function buildQuickForm() {
     //get the frequency units.
     $this->_freqUnits = CRM_Core_SelectValues::getRecurringFrequencyUnits();
 
-    $numericOptions = CRM_Core_SelectValues::getNumericOptions(0, 30);
-
     //reminder_interval
-    $this->add('select', 'start_action_offset', ts('When'), $numericOptions);
+    $this->add('number', 'start_action_offset', ts('When'), array('style' => 'width:5em'));
```
Comment (colemanw) [inline CRM/Admin/Form/ScheduleReminders.php]: can you add a css class instead of an inline style? Like `'class' => 'six'` ought to do it.

Comment (colemanw) [inline CRM/Admin/Form/ScheduleReminders.php]: Also add property `'min' => 1`

Comment (konadave) [inline CRM/Admin/Form/ScheduleReminders.php]: I did add 'min' => 1, but later noticed the select controls actually have 0-30, so changed min to zero.

**Comment by colemanw** [comment]
Ok then `positiveInteger` is probably the wrong validation rule.

**Comment by konadave** [comment]
`positiveInteger` validates zero.

**Comment by colemanw** [comment]
Cool.

**Pull request merged**